packages/block-editor/src/components/inserter/tips.js

```diff
@@ -28,7 +33,79 @@ const globalTips = [
 	__( "Change a block's type by pressing the block icon on the toolbar." ),
 ];
 
-function Tips() {
+const contextualTips = {
+	css: createInterpolateElement(
+		__(
+			'CSS - You can visit the <a>the Customizer</a> to edit the CSS on your site.'
+		),
+		{
+			a: <a href="/customize.php?autofocus[section]=custom_css" target="_blank"/>
+		}
+	),
+
+	theme: createInterpolateElement(
+		__(
+			'theme - You can visit the <a>theme directory</a> to select a different design for your site.'
+		),
+		{
+			a: <a href="/themes.php" target="_blank"/>
+		}
+	),
+
+	plugin: createInterpolateElement(
+		__(
+			'plugin - You can visit the <a>plugin directory</a> to install additional plugins.'
+		),
+		{
+			a: <a href="/plugin-install.php" target="_blank"/>
+		}
+	),
+
+	header: createInterpolateElement(
+		__(
+			'header - You can visit the <a>the Customizer</a> to edit your logo and site title.'
+		),
+		{
+			a: <a href="/customize.php?autofocus[section]=title_tagline" target="_blank"/>
+		}
+	),
+
+	colors: createInterpolateElement(
+		__(
+			'colors - You can visit the <a>the Customizer</a> to edit the colors on your site.'
+		),
+		{
+			a: <a href="/customize.php?autofocus[section]=colors" target="_blank"/>
+		}
+	),
+};
```

Comment:
While I like the PR, I think most of these contexts and tips are WP-specific while the `block-editor` package is not. (Imagine an editor used to write comments, or outside WP entirely).

We'd have to find a way to "register" these tips on "edit-post" maybe.

Reply:
what do you think about adding a slot there?

Reply:
I feel this is more something for a "registration APi". We also have an issue about the possibility to register block-specific tips, so it can scale to absorb that too.

Reply:
@davemart-in FYI, as this will increase the scope of #20196

Reply:
> We also have an issue about the possibility to register block-specific tips, so it can scale to absorb that too.

This one https://github.com/WordPress/gutenberg/issues/17091?

Reply:
I agree this should be on a register based API. It also needs to handle user permissions, since access to many of those admin views is not a given for all roles.